lib/rubocop/cop/style/space_around_keyword.rb

```diff
@@ -133,6 +133,7 @@ def check(node, locations, begin_keyword = DO)
             range = node.loc.public_send(loc)
             next unless range
 
+            # node.children[0].source
```

Comment:
Yes it is. It was a note to myself about a possible solution. Thanks for catching this.